backend/.env.sample

```diff
@@ -21,12 +22,16 @@ GOOGLE_MAP_API_KEY=
 # will be deprecated
 DISFACTORY_IMGUR_CLIENT_ID=your_imgur_id
 DISFACTORY_BACKEND_MEDIA_ROOT="./images/"
-DISFACTORY_BACKEND_DOMAIN="https://api.disfactory.tw/"
-DISFACTORY_FRONTEND_DOMAIN="https://disfactory.tw/"
+DISFACTORY_BACKEND_DOMAIN="http://localhost/"
+#DISFACTORY_BACKEND_DOMAIN="http://ohshown.site/"
+OHSHOWN_FRONTEND_DOMAIN="http://localhost/"
+#OHSHOWN_FRONTEND_DOMAIN="http://ohshown.site/"
 
 DISFACTORY_PGDATA_PATH=/tmp/disfactory_data/
 DISFACTORY_PGDATA_BACKUP_PATH=/tmp/disfactory_db_dump/
 
 DISFACTORY_CADDY_STATIC_DIR=/tmp/disfactory/static
 
-COMPOSE_PROJECT_NAME=staging
+COMPOSE_PROJECT_NAME=localdev
```

Comment:
In my opinion, `localdev` is a better name for `staging` when developing locally.